packages/front-end/src/app/components/EGLabView.vue

```diff
@@ -635,7 +668,7 @@
       <!-- HealthOmics Pipelines tab -->
       <div v-if="item.key === 'omicsWorkflows'" class="space-y-3">
         <EGTable
-          :row-click-action="viewRunOmicsWorkflow"
+          :row-click-action="(row) => viewRunDetails(row)"
```

Comment:
this isn't right, `viewRunDetails` is for viewing a Run, but this action is on click of an `Omics Workflow` - it should stay as `viewRunOmicsWorkflow` which currently just toasts that it's not implemented yet.